mobile/src/main/java/com/github/shadowsocks/App.kt

```diff
@@ -163,6 +163,15 @@ class App : Application() {
         }
 
         updateNotificationChannels()
+
+        val nightMode = DataStore.nightMode
+        when (nightMode) {
+            Key.nightModeSystem -> AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_FOLLOW_SYSTEM)
+            Key.nightModeAuto -> AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_AUTO)
+            Key.nightModeOff -> AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO)
+            Key.nightModeOn -> AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES)
+            else -> AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO)
+        }
```

Comment:
I don't think @madeye likes to have a configuration for UI options. I am okay with this change.